whisper/whisperv6/peer.go

```diff
@@ -38,6 +39,7 @@ type Peer struct {
 	powRequirement float64
 	bloomFilter    []byte
 	fullNode       bool
+	mu             sync.Mutex
```

Comment:
it is good practice to put mutex above protected variables, and rename it accordingly. maybe `bloomMu`? cause fullNode is also about bloom filters